rtl_tests.c

```diff
@@ -211,8 +211,80 @@ void test_RtlAssert(){
     /* FIXME: This is a stub! implement this function! */
 }
 
+// FIXME: RtlCaptureContext also sets Ebp, Eip, and Esp in the context record.
+// However, without doing some math there isn't a good way to know what Ebp, Esp, and Eip were
+// set to in the caller of RtlCaptureContext. And I'm too lazy to figure out that code right now.
```

Comment:
`FloatSave` isn't handled at all. But is not mentioned here, or in the PR description?

Reply:
In my kernel dump of RtlCaptureContext, the Floating save area is not touched. CONTEXT addresses 0x21C, 0x218, 0x214, 0x210, 0x20C, 0x208, 0x228, 0x234, 0x22C, 0x224, 0x220, and 0x230 are set in the function which line up with the addresses for the flags and general purpose registers.

Reply:
Probably worth adding a comment here, or even in nxdk.